### PR TITLE
Aastha_Assignment_33_Done_30mins_2

### PR DESCRIPTION
**Pull request by AasthaJJun20**
-> Assignment-33 Done (created new branch since the old one had issues)
->PFB output:
ItemIDFromURL-output

### CODE REVIEW

**Review by NikhilMJun20 — APPROVED** [review]
Code and logic look good to me.
I am approving the PR.

File: src/aasthaJJun20/ItemIDFromURL.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+public class ItemIDFromURL {
+	void findItemIDFromURL(String url) {
+		System.out.println("\nString input : " + url);
+		String temp = "";
+		try {
+			temp = url.substring(url.indexOf("itemId"));
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/ItemIDFromURL.java:20]: try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

File: src/aasthaJJun20/ItemIDFromURL.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		} catch (StringIndexOutOfBoundsException stringIndexOutOfBoundsException) {
+			System.out.println("No item id present");
+			return;
+		}
+		String itemId = "";
+		for (int index = 0; index < temp.length(); index++) {
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/ItemIDFromURL.java:26]: if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Comment (AasthaJJun20) [reply src/aasthaJJun20/ItemIDFromURL.java:26]: Thanks Krishna!!

**Pull request merged**